website/src/pages/jobs.tsx

```diff
@@ -43,7 +43,7 @@ export default class JobsPage extends React.Component<any, any> {
                                 Join us on our mission to make it so everyone, in every community, in every country, and
```

Comment:
Should we delete this page entirely and have this url go directly to `/company/careers`?

Reply:
I left it thinking that it might be bookmarked or linked in a bunch of places, but if we just made it a redirect that could work. @aileenrose - what do you think is best to do here?